go/libkb/session.go

```diff
@@ -15,6 +15,8 @@ import (
 
 type SessionReader interface {
 	APIArgs() (token, csrf string)
+	IsLoggedIn() bool
```

Comment:
Can you provide more color as to what a `SessionReader` is and who implement is?